_partials/_install-self-hosted-docker-based.mdx

```diff
@@ -19,15 +18,13 @@
 
     If you are running multiple container instances remember to change the port each Docker instance runs on.
 
-
  1. **Connect to a database on your PostgreSQL instance**
 
-   In PostgreSQL, the default user and database are both `postgres`. To use a
-   different database, set `<database-name>` to the name of that database:
+   In PostgreSQL, the default user and database are both `postgres`. The default command for this Docker container is:
 
-   ```bash
-   psql -d "postgres://<username>:<password>@<host>:<port>/<database-name>"
-   ```
+    ```bash
+    psql -d "postgres://postgres:password@localhost/postgres"
```

Comment:
Is the password literally "password"? I can't just copy and paste this command, so something else is missing. (`psql: error: connection to server at "localhost" (::1), port 5432 failed: FATAL:  role "postgres" does not exist`)

Reply:
If you run the command in the previous step for the *pg16 image this works (on my machine :-( ). I have added an explanation.